staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```diff
@@ -264,7 +258,8 @@ func (o *ScaleOptions) RunScale() error {
 		}
 
 		return o.PrintObj(info.Object, o.Out)
```

Comment:
Ooops.. Now that we are iterating `infos`, we could not simply return inside the for loop. 

As an alternative, we could wrap the original visit logic inside a new funcion, for instance:
```
visit := func(info *resource.Info) error {
  ...
}
for _, info := range infos {
  err := visit(info)
  if err != nil {
    return err
  }
}
```